text/lang-detection/scrape.jl

```diff
@@ -7,17 +10,18 @@ pages = Dict(
   :es => ["Wikipedia", "Chorizo", "Historia_de_Barcelona", "Espana", "Las_Vegas_Strip", "Judaismo"],
   :da => ["Wikipedia", "H.C._Andersen", "L.A._Ring", "Jiangxi", "NATO", "Thomas_Edison", "Bangladesh"])
 
-rawpage(url) = parsehtml(String(HTTP.get(url).body)).root
-
 function innerText(dom)
-  text = IOBuffer()
+  text = ""
   for elem in PreOrderDFS(dom)
-    elem isa HTMLText && print(text, elem.text)
+        if elem isa HTMLText
+            text = string(text, elem.text)
+        end
```

Comment:
We could perhaps retain `IOBuffer` with something like `write(text, elem.text)`?